website/blog/2020-04-21-wavemaker-openapi-import.md

```diff
@@ -0,0 +1,61 @@
+<!-- truncate -->
+
+![OpenApi](/learn/img/OpenApi_Logo.png)
+
+## Introduction 
+
+OpenAPI is a broadly adopted industry standard for describing modern APIs. The official release of OpenApi Specification 3.0 was marked as a special one as it was the first release since it was donated to the OpenAPI initiative by SmartBear Software and was renamed from Swagger specification to OpenAPI specification.
+
+First let’s be clear on how OpenAPI is different from Swagger. The easiest way to understand the difference is:
+
+    OpenAPI = Specification
+    Swagger = Tools to implement the specification
+
+
+The development of the specification is fostered by the OpenAPI initiative, which involves organizations from different areas of the tech world including Microsoft, Google, IBM, Oracle, ebay, etc. More information about the OpenAPI initiative can be found [here](https://www.openapis.org/). Swagger has been the major contributor towards the community and provides multiple tools to build and test OpenAPI compliant APIs. More information on swagger can be found [here](https://swagger.io/).
+
+  
+## OpenAPI specification
+The OpenAPI Specification (OAS) defines a standard, programming language-agnostic interface description for REST APIs, which allows both humans and computers to discover and understand the capabilities of a service without requiring access to source code, additional documentation, or inspection of network traffic. When properly defined via OpenAPI, a consumer can understand and interact with the remote service with a minimal amount of implementation logic. Similar to what interface descriptions have done for lower-level programming, the OpenAPI Specification removes guesswork in calling a service. 
+
+A document (or set of documents) that defines or describes an API. An OpenAPI definition uses and conforms to the OpenAPI Specification. The specification can be found [here](http://spec.openapis.org/oas/v3.0.3).
+
+:::note
+As of release 10.4, WaveMaker supports OAS 2.0. Support for OSA 3.0 is being worked upon. Keep an eye on this space for more updates.
+:::
+
+  
+## Advantage of working with OpenAPIs
+Since an OpenAPI definition document follows a standard OpenAPI specification, the document can be used by documentation generation tools to display the API, code generation tools to generate servers and clients in various programming languages, testing tools, and many other use cases.
+
+WaveMaker extends this advantage to integrate REST endpoints in a Web/Mobile application. Integrating a REST endpoint with any of the 100+ UI widgets offered by WaveMaker is simple and straightforward without writing a single line of code.
```

Comment:
can we start with this statement. Then go into what OpenAPI/Swagger is.
Something like:
WaveMaker generates UI for create, list, edit, delete actions for REST APIs that conform to OpenAPI standard.  You can also use any of the 100+ UI widgets and bind them to the REST API. OpenAPI is opensource standard ......

Reply:
I thought it would be better to set the context first with OpenApi spec in general terms, then it is easier to understand integration with any tool like WaveMaker

Reply:
Changed the content a bit

Reply:
Start with WM value prop before truncation, because this is the hook for the reader to spend more time. It is ok to assume that reader is aware what OpenAPI is.

Reply:
Done